### ISSUE TITLE
Settings: Sub-chain - data table 2/4

### ISSUE THREAD

**Issue by crsfalzon** [SimplyVC/panic#225]
### User Story
As a node operator, I want to read/update/delete my sub-chain setup, so this can be adjusted to meet my operational needs. 

### Description
The scope of this ticket is the implementation of the data table. The implementation of the enable/disable, edit, and delete functionality is **not** in scope.

![data table](https://user-images.githubusercontent.com/89395527/171136661-bcdef340-2d15-4f75-8715-ea322e390371.PNG)

### Requirements

- Columns: Name, Base-chain, Enabled, Actions
- The Enabled column will consist of checkboxes
- The Actions column will consist of edit and delete buttons
- The table is populated with all the previously set sub-chains (unless these were deleted)
- The node operator is allowed to filter by name and base-chain

### Blocked by
Not dependent on another ticket to be tackled.

### Acceptance criteria

**Scenario**: The node operator is on the configuration/sub-chains after having selected Chains from the Settings dropdown #223
**Given**: The node operator wants to read/update/delete a sub-chain
**When**: On _/configuration/sub-chains_ 
**Then**: Make available a data table showing name, base-chain, enabled (checkbox), and actions (edit/delete) for every sub-chain
